cosmos/providers/databricks/notebook.py

```diff
@@ -124,7 +126,10 @@ def monitor_databricks_job(self):
 
     def _get_current_databricks_task(self, runs_api):
         return {
-            x["task_key"]: x for x in runs_api.get_run(self.databricks_run_id)["tasks"]
+            x["task_key"]: x
+            for x in runs_api.get_run(self.databricks_run_id, version=JOBS_API_VERSION)[
+                "tasks"
+            ]
```

Comment:
Do we have any unit tests to cover this change?

Reply:
Those were missing, I have added them now.